packages/website-frontend/test/unit/services/auth.service.spec.ts

```diff
@@ -0,0 +1,91 @@
+import { describe, it, expect, beforeEach, vi, afterEach } from "vitest";
+import createFetchMock, { FetchMock } from 'vitest-fetch-mock';
+
+import { AuthenticateResponse } from "@stryker-mutator/dashboard-contract";
+import { AuthService } from "../../../src/services/auth.service";
+import { SessionStorageService } from "../../../src/services/session-storage.service";
+
+describe(AuthService.name, () => {
+  let authService: AuthService;
+  let mockSessionStorageService: SessionStorageService;
+  let fetchMock: FetchMock;
+
+  beforeEach(() => {
+    vi.restoreAllMocks();
+
+    mockSessionStorageService = {
+      getItem: vi.fn(() => 'foo-token'),
+      setItem: vi.fn(),
+      removeItem: vi.fn(),
+    };
+    authService = new AuthService(mockSessionStorageService);
+
```

Comment:
Why not use a real session storage here?

Reply:
Good question, these might be remnants from when I was using happy-dom.. though I’m not sure 🤔